cargo/cargo_build_script.bzl

```diff
@@ -36,11 +47,21 @@ def _cargo_build_script_run(ctx, script):
         ],
     )
 
-    ctx.actions.run(
-        executable = ctx.executable._cargo_build_script_runner,
-        arguments = [script.path, env_out.path, flags_out.path],
-        outputs = [out_dir, env_out, flags_out],
+    cmd = ""
```

Comment:
Maybe comment that part to explain for future reader?

Reply:
done